weasyl/controllers/settings.py

```diff
@@ -411,6 +411,25 @@ def control_apikeys_post_(request):
 
 
 @login_required
+def control_editsearchtagblacklist_get_(request):
+    # Get the user's blacklist tag settings and render to the template
+    return Response(define.webpage(request.userid, "control/edit_searchtagblacklist.html", [
```

Comment:
Nitpick: could you make the template arguments tuples instead of lists, please? (I know, it’s inconsistent with everything else, but it’s correct.)